### ISSUE TITLE
When I execute the ./app install command, I got a error

### ISSUE THREAD

**Issue by GuofaPython** [phase1geo/Minder#468]
Directory already configured.

Just run your build command (e.g. ninja) and Meson will regenerate as necessary.
If ninja fails, run "ninja reconfigure" or "meson --reconfigure"
to force Meson to regenerate.

If build failures persist, run "meson setup --wipe" to rebuild from scratch
using the same options as passed when configuring the build.
To change option values, run "meson configure" instead.
[2/3] Compiling C object 'com.github.phase1geo.minder@exe/meson-generated_src_Node.c.o'.
FAILED: com.github.phase1geo.minder@exe/meson-generated_src_Node.c.o 
cc -Icom.github.phase1geo.minder@exe -I. -I.. -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include -I/usr/include/cairo -I/usr/include/pixman-1 -I/usr/include/uuid -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/gee-0.8 -I/usr/include/granite -I/usr/include/at-spi2-atk/2.0 -I/usr/include/at-spi-2.0 -I/usr/include/dbus-1.0 -I/usr/lib/x86_64-linux-gnu/dbus-1.0/include -I/usr/include/gtk-3.0 -I/usr/include/gio-unix-2.0 -I/usr/include/pango-1.0 -I/usr/include/fribidi -I/usr/include/harfbuzz -I/usr/include/atk-1.0 -I/usr/include/gdk-pixbuf-2.0 -I/usr/include/libmount -I/usr/include/blkid -I/usr/include/libxml2 -I/usr/include/gtksourceview-4 -I/usr/include/libhandy-1 -I/usr/include/json-glib-1.0 -fdiagnostics-color=always -pipe -D_FILE_OFFSET_BITS=64 -w -g '-DGETTEXT_PACKAGE="com.github.phase1geo.minder"' -pthread -include config.h -MD -MQ 'com.github.phase1geo.minder@exe/meson-generated_src_Node.c.o' -MF 'com.github.phase1geo.minder@exe/meson-generated_src_Node.c.o.d' -o 'com.github.phase1geo.minder@exe/meson-generated_src_Node.c.o' -c 'com.github.phase1geo.minder@exe/src/Node.c'
../src/Node.vala: In function ‘node_instance_init’:
../src/Node.vala:151:11: error: expected expression before ‘{’ token
  151 |   private   RGBA?        _link_color = {1.0, 1.0, 1.0, 1.0};
      |           ^
ninja: build stopped: subcommand failed.
Unable to build project, please review log

**Comment by phase1geo** [comment]
There shouldn't be a compile issue here.  What version of valac is installed on your system?

**Comment by phase1geo** [comment]
Yes, there should definitely not be an issue compiling with that version of valac.  I'm not sure what the problem is for you.  Out of curiosity, can you try ./app clean followed by ./app install?

**Comment by GuofaPython** [comment]
> Yes, there should definitely not be an issue compiling with that version of valac. I'm not sure what the problem is for you. Out of curiosity, can you try ./app clean followed by ./app install?

According to your suggestions I have executed to complete this command but the result is to get the same error. Before executing your suggestion I tried to modify the code, changed **_link_color = {1.0, 1.0, 1.0, 1.0} to **_link_color = null, and executed **./app install** command, successfully built Minder, but I could not open it successfully, when I executed command **./app run**, or used the mouse click in the launcher, it only appeared on the screen for a moment and then automatically closed.

**Comment by GuofaPython** [comment]
> I would not assign this to null. If you want to try a code change, you could try moving this default assignment to all of the constructors of the Node class and then just leave _link_color assigned to nothing on the failing line.

emm, modifying a lot of code is still too difficult for me...

**Comment by GuofaPython** [comment]
> But, this code should not need to be modified. I constantly compile Minder without issue -- I've compiled it with various versions of 0.48.x and newer without issues with this syntax.

Okay I'll continue to try to build mider on my utbuntu20.4 system, and I really hope it succeeds, as the version of the app installed **sudo apt install minder** command is too low and some of it complicates my job.

**Comment by ookiineko** [comment]
![image](https://user-images.githubusercontent.com/103345643/163707684-0ab9070b-6c07-46ed-9f72-72e2a82b9b18.png)

Hi, I got the same issue here, and I am able to solve it by upgrading my valac compiler to the version in Ubuntu Jammy, and it runs well.

It is definitely easy to upgrade valac, just downloading the depended packages from [Launchpad](https://launchpad.net/ubuntu) and install them manually.

I also installed some other outdated/missing runtime/build dependencies from Ubuntu Hirsute/Impish/Jammy for building it xDD

**Comment by ookiineko** [comment]
After Ubuntu Focal, it seems they renamed the `libgdk-pixbuf2.0-0` package to `libgdk-pixbuf-2.0-0`, which would prevent the above mentioned dependencies from being installed, which I fixed by creating a dummy package using equivs. [Here](https://gist.github.com/ookiineko/3669f04ad21818863e6621d83db2bec8) is the control file I used.

**Comment by prish-collab** [comment]
I got the exact same error as @GuofaPython.. can any one explain what I need to do to make this code running?? I am running in on Ubuntu 20.04..

**Comment by farooqkz** [comment]
Can this be closed now?
